tests/integration/testdata/sync/infra/before/Python/function/app.py

```diff
@@ -0,0 +1,24 @@
+import json
+from layer_method import layer_method
+
+import requests
+
+
+def lambda_handler(event, context):
+    """Sample pure Lambda function that returns a message and a location"""
+
+    try:
+        ip = requests.get("http://checkip.amazonaws.com/")
```

Comment:
This can be slow and is an external dependency. Is this needed or a simple message will be sufficient?

Reply:
We want to have a simple test of such dependencies, even though a simple message is sufficient in terms of testing the lambda function, increasing the complexity a bit is helpful for the sake of testing function with dependencies.

Reply:
What dependency are we testing here?
I agree that adding complexity and relevant dependencies to tests will increase coverage, but here, we are testing whether Lambda can make requests to and get IP back from `checkip.amazonaws.com` which does not involve SAM CLI.

Reply:
If the imports failed then the lambda would fail since the dependency wasn't included properly

Reply:
In that case, can we switch to another library like `numpy` which doesn't require rely on a external remote website? This way we can minimize chances of having flaky tests.

Reply:
I changed external libraries with `numpy` and `ruby-statistics` so that there won't be an external call during tests